articles/9-tips-for-student.md

```diff
@@ -0,0 +1,171 @@
+
+#### Useful resources
+🔗 [Making out the most out of your PFE internship](https://www.youtube.com/watch?v=uhLeoSeZxz8): Geeksblala's podcast episode about PFE interships.
+🔗 [Computer Science Morocco Community](https://www.youtube.com/@cs-morocco): Moroccan community for people interested in competetive programming and software engineering in general.
+🔗 [Build your own X](https://github.com/codecrafters-io/build-your-own-x): Collection of resources on how to build common software from scratch.
+
+## 5. Use school projects to your advantage
+
+School projects are a fantastic opportunity to learn. Even better if they’re fun and interesting.
+
+My advice is always to try to make your school projects work for you. 
+
+First, if possible, negotiate the topic to align with your interests. 
+
+Then focus on what is offers you the most value and delegate the less interesting parts to your teammates, open-source code, or even no-code tools.
+
+Finally, publish the project and make it available for everyone to check. This will help your brand and help you connect to link-minded people from the community.
+
+All of this not only helps you stay engaged but also differentiates your work, even if it’s technically the same project as everyone else’s.
+
+#### Useful resources
+🔗 [SaaS Lifecycle in Morocco](https://www.youtube.com/watch?v=0R7eUTy51UQ): Geeksblabla's episode about SaaS.
+🔗 [No Code tools directory](https://www.nocode.tech/tools): List of no-code tools
+🔗 [10 Tips To Host Your Web Apps For Free](https://blog.patricktriest.com/host-webapps-free/): Good article about hosting web apps for free.
+
+## 6. Contribute to open source
+This is one of the most overrated yet invaluable pieces of advice I got as a student.
+
+Back then, I thought: How can someone who doesn’t even know if HTML is a programming language contribute to open source?
+
+To be honnest, that was on me. I didn’t understand what “contributing to open source” really meant. Contributing to OSS isn’t always about writing code.
+
+Start by using open-source tools. This alone is a form of contribution. Then, engage with the community: read the code, ask questions, open issues, suggest ideas, or join discussions on forums and Discord.
+
+Even without contributing a single line of code, this process will teach you how software works, help you build a name for yourself in the community, and prepare you to contribute code when you’re ready.
+
+> 💡 Tip: Building your own open-source project also counts as OSS contribution. 😉
+
+#### Useful resources
+🔗 [Open Source & Hacktoberfest 21](https://www.youtube.com/watch?v=T0ebJvUiDG4): A Geeksblabla episode about Open Source.
+🔗 [Moroccan Open Source projects](https://awesome-morocco.dev/open-source): List of Moroccan Open Source projects.
+🔗 [Open Source Guide](https://opensource.guide/): Various guides on how to contribute to open source.
```

Comment:
Will these be rendered each per line?

Reply:
I think so,  will fix them if it's not the case